### ISSUE TITLE
Don't render `classDef` construction of mermaid-syntax

### ISSUE THREAD

**Issue by GreenRobot777** [benweet/stackedit#1539]
`StackEdit v5.14.5`

```mermaid
graph LR
    nodeId1(sdf) --> nodeId2(asdf)
    nodeId2 --> nodeId3
    nodeId3(sdf) --> nodeId4(asdf)
    classDef trigger fill:#fFF2,stroke:#333,stroke-width:1px
    classDef sms fill:#FFBBCC,stroke:#333,stroke-width:1px
    class nodeId1,nodeId2 trigger;
    class nodeId3,nodeId4 sms;

```
how it should be:
![image](https://user-images.githubusercontent.com/14993551/63673892-8b618700-c80f-11e9-81ff-7a4098e5fcd8.png)

how is rendering in Stackedit.io:
![image](https://user-images.githubusercontent.com/14993551/63673952-a7652880-c80f-11e9-952a-d05c3fbdd824.png)

**Comment by sdc4** [comment]
Would also be my question, if a solution or workaround exists.

**Comment by benweet** [comment]
Fixed with v5.14.10.
